pkgs/development/libraries/gsignond/plugins/lastfm.nix

```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
   PKG_CONFIG_GSIGNOND_GPLUGINSDIR = "${placeholder "out"}/lib/gsignond/gplugins";
 
   meta = with stdenv.lib; {
-    description = "Plugin for the Accounts-SSO gSignOn daemon handles the Last.FM credentials.";
+    description = "Plugin for the Accounts-SSO gSignOn daemon handles the Last.FM credentials";
```

Comment:
`Last.FM Plugin for the gSignOn daemon`
or
`This plugin for the Accounts-SSO gSignOn daemon handles Last.FM credentials`